.github/CHANGELOG.md

```diff
@@ -26,7 +27,7 @@
 
 This release contains contributions from (in alphabetical order):
 
-Nariman Saadatmand
+[Joost Bus](https://github.com/soosub), [Nariman Saadatmand](https://github.com/nariman87)
```

Comment:
Check with @nariman87 if he'd like to have his handle in here.

Reply:
```suggestion
[Joost Bus](https://github.com/soosub), Nariman Saadatmand
```